theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -1,3 +1,11 @@
+def debian_release_to_version = [
+    'buster': 'deb10',
+    'bullseye': 'deb11',
+    'bookworm': 'deb12',
```

Comment:
should this say `debian` instead of `deb`?

Reply:
That feels cleaner, and if you make that change and want to merge go for it.

Reply:
Will do (tomorrow)

Reply:
done, and added 13 and 14 codenames as they are already known